Emby.Server.Implementations/LiveTv/EmbyTV/EmbyTV.cs

```diff
@@ -1644,22 +1611,20 @@ private void DeleteLibraryItemForTimer(TimerInfo timer)
 
             if (libraryItem != null)
             {
-                _libraryManager.DeleteItem(libraryItem, new DeleteOptions
-                {
-                    DeleteFileLocation = true
-
-                }, true);
+                _libraryManager.DeleteItem(
+                    libraryItem,
+                    new DeleteOptions
+                    {
+                        DeleteFileLocation = true
+                    },
+                    true);
             }
             else
             {
-                try
+                if (File.Exists(timer.RecordingPath))
```

Comment:
I actually don't like when you remove exception handling, what if file exists but we cannot delete it for whatever reason (no permissions, read-only file system, i/o failure)?

Reply:
The caller already catches it
```cs
                try
                {
                    DeleteLibraryItemForTimer(timer);
                }
                catch (Exception ex)
                {
                    _logger.LogError(ex, "Error deleting recording");
                }
```